### DISCUSSION TITLE
How do I download attachments from issues?

### DISCUSSION THREAD

**Discussion by Centurius** [sumoheavy/jira-ruby#449] [Q&A]
I am trying to donwload the attachment that comes with this issue, but it always complains about an undefined method:

  issue = client.Issue.find('PRIVACYEXT-269')
  issue.attachments.first.download_file

undefined method `download_file' for an instance of JIRA::Resource::Attachment

it looks like i have the correct object, but i am stuck here, there ist jsut an example for uploading, not downloading:
#<JIRA::Resource::Attachment:740

**Comment by bobbrodie** [comment]
Hi @Centurius this is something that should be included in the 3.0 release, which is upcoming. I plan on pushing 3.0.0-alpha1 on Monday, Aug 12 after merging some rubocop fixes.

There are a few parts to it:
* https://github.com/sumoheavy/jira-ruby/pull/422
* https://github.com/sumoheavy/jira-ruby/issues/443
* https://github.com/sumoheavy/jira-ruby/pull/444

Once released you will be able to use the included methods: `download_file` and `download_contents`.
